documentation/tarron_notes.md

```diff
@@ -1,5 +1,15 @@
 # Tarron's Notes
 
+## Questions
+
+- [ ] What does this notation `(rootQuery, { planId }, context)` mean in lambda function declaration
+- [ ] Add Stripe CustomerID on space center?
+- [ ] Can a single plan be tied to multiple products?
+
+## ToDo
+
+- [ ] Tie our database users to firebase auth users. This will need to be added to our generate script
```

Comment:
Does that match the id of the "users" collection in the firestore database?

Reply:
Yes, because that's the ID we use when we define the user object in the 'users' collection. The auth user ID and the 'users' collection ID are one and the same.